data/sql/updates/pending_db_world/bonelasher-oob.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM `creature` WHERE `guid` IN (66458, 66459);
```

Comment:
Missing AND `id1`=XXXX

Reply:
Update [the wiki](https://www.azerothcore.org/wiki/sql-standards) with that info, please.

Reply:
No no I meant id1 😄 
There are hundreds of creatures with curhealth=5715, and thousands with map=530

```
DELETE FROM `creature` WHERE `guid` IN (66458, 66459) AND `id1`=18470;
```

Reply:
And there are most definitely > 2 creatures with an id1 of 18470, I really don't see the point of specifying further than a keyed column where duplicates aren't possible.

Reply:
It's been a longstanding practice for quite a while

![image](https://user-images.githubusercontent.com/100873791/210021035-ba3fc106-890a-45d3-b5b1-ccc969dd76c5.png)

Reply:
> It's been a longstanding practice for quite a while

I would not consider under two months to be "quite a while". But even still, it's not in the standard operating procedure on the wiki with regards to SQL queries, so I still wouldn't call it a standard practice.

Reply:
That's just an example I found. I've always been taught to add GUIDs and IDs together.

Reply:
Well for people like me, these standards need to be on the wiki. I can only scan through so many issues/PRs I have nothing to do with.